docs/content/en/docs/architecture/components/scheduler/_index.md

```diff
@@ -1,13 +1,48 @@
 ---
-title: Keptn Lifecycle Scheduler
-linktitle: Scheduler
-description: Basic understanding of the Keptn Scheduler
+title: Keptn integration with Scheduling
+linktitle: Scheduler and Scheduling Gates
+description: Basic understanding of how Keptn integrates with Kubernetes Pod Scheduling
 weight: 80
 cascade:
 ---
 
+Keptn needs to integrate with Kubernetes scheduling to block
+the deployment of applications that do not satisfy Keptn defined pre-deployment checks.
+
+On Kubernetes versions older and including 1.26, Keptn uses a Scheduler to block application deployment.
 The **Keptn Scheduler** is an integral component of Keptn that orchestrates
 the deployment process.
+
+If the Keptn helm chart value `schedulingGatesEnabled` is set to to `true`, and Keptn is running on a Kubernetes version greater than 1.26,
+Keptn does not install a scheduler plugin.
+Instead, it uses
+the [Pod Scheduling Readiness K8s API](https://kubernetes.io/docs/concepts/scheduling-eviction/pod-scheduling-readiness).
+
+## Keptn Scheduling Gates for K8s 1.27 and above
+
+When you apply a workload to a K8s cluster, the Mutating Webhook checks for annotations on Pods to
+see if it is annotated with
+[Keptn specific annotations](../../implementing/integrate/#basic-annotations).
+If the annotations are present, the Webhook adds a gate to the Pod called `keptn-prechecks-gate`,
+this is a spec that tells the Kubernetes scheduling framework to wait for Keptn's checks before assigning the pod to a
+node.
+
+For instance a pod gated by keptn will look like the following:
+{{<embed path="/docs/assets/scheduler-gates/gated.yaml">}}
+
+The **WorkloadInstance CRD** contains information about the `pre-deployment` checks that
+need to be performed before the Pod can be scheduled.
+If the `pre-deployment` checks have finished successfully, the WorkloadInstance Controller removes the gate from the
+Pod.
+The scheduler can then allow the Pod to be scheduled to a node.
+If the `pre-deployment` checks have not yet finished, the gate will stay and the Pod will remain pending.
+When removing the gate, the WorkloadInstance controller also adds the following annotation, so that if there is an
+update of the spec the Pod will not be gated again:
```

Comment:
```suggestion
When removing the gate, the WorkloadInstance controller also adds the following annotation so that,
if the spec is updated,
the Pod is not gated again:
```